types/noloopfor/index.d.ts

```diff
@@ -0,0 +1,8 @@
+// Type definitions for noloopfor 2.0
+// Project: https://github.com/david-fernando/noLoopFor
+// Definitions by: David Fernando <https://github.com/david-fernando/>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+export function loop(value: number, target: number, callback: any): any;
+export function loopStep(value: number, target: number, step: number, callback: any): any;
+export function decrementLoop(value: number, target: number, callback: any): any;
+export function decrementStep(value: number, target: number, step: number, callback: any): any;
```

Comment:
Seems like these should all take actual callback types

```suggestion
export function loop(value: number, target: number, callback: (index: number) => void): any;
export function loopStep(value: number, target: number, step: number, callback: (index: number) => void): any;
export function decrementLoop(value: number, target: number, callback: (index: number) => void): any;
export function decrementStep(value: number, target: number, step: number, callback: (index: number) => void): any;
```